index.html

```diff
@@ -54,119 +62,123 @@ <h1 class="intro height">I’m Kartikay Glad to see you!</h1>
   </section>
   <section>
     <div class="grid-container">
-      
       <div class="card">
         <div class="card-element">
-          <div class="img-container">
-            <img class="work-image" src="./Images/tonic-one.png" alt="work-Image" />
-          </div>
-          <h2 class="card-title">Tonic</h2>
-          <div class="card-section">
-            <p class="para-one">CANOPY</p>
-            <i class="fa fa-circle"></i>
-            <p class="para-two">Back End Dev</p>
-            <i class="fa fa-circle"></i>
-            <p class="para-two">2015</p>
+          <img class="work-image desktop-image" src="./Images/tonic-one.svg" alt="work-Image" />
+          <img class="work-image mobile-image" src="./Images/tonic-one.png" alt="work-Image" />
+          <div class="card-content-container">
+            <h2 class="card-title">Tonic</h2>
+            <div class="card-section">
+              <p class="para-one">CANOPY</p>
+              <i class="fa fa-circle"></i>
+              <p class="para-two">Back End Dev</p>
+              <i class="fa fa-circle"></i>
+              <p class="para-two">2015</p>
+            </div>
+            <p class="card-body">
+              A daily selection of privately personalized reads; no accounts
+              or sign-ups required.
+            </p>
+            <ul class="tech-stack">
+              <li><button class="tech-items" type="button">html</button></li>
+              <li><button class="tech-items" type="button">css</button></li>
+              <li>
+                <button class="tech-items" type="button">javascript</button>
+              </li>
+            </ul>
+            <a class="card-button" href="#">See Project</a>
           </div>
```

Comment:
- [ ] Please make sure the text inside this button is centered, as in the template.

![](https://i.imgur.com/oWgRDP4.png)

Reply:
Thanks for the remark. I will fix these.